apps/frontend/src/utilities/aws_util.ts

```diff
@@ -88,43 +85,43 @@ export function derive_mfa_serial(user_access_token: string): string | null {
  * Yields the AWS error on failure.
  */
 export async function get_session_token(
-  access_token: string,
-  secret_key: string,
+  accessToken: string,
+  secretKey: string,
   duration: number,
-  mfa_info?: MFA_Info
+  mfaInfo?: MFAInfo
 ): Promise<Auth> {
   // Instanciate STS with our base and secret token
   const sts = new STS({
-    accessKeyId: access_token,
-    secretAccessKey: secret_key
+    accessKeyId: accessToken,
+    secretAccessKey: secretKey
   });
 
   // Get the user info
-  const wip_info: Partial<AuthInfo> = {};
+  const wipInfo: Partial<AuthInfo> = {};
   await sts
     .getCallerIdentity({})
     .promise()
     .then((success) => {
-      wip_info.user_account = success.Account!;
-      wip_info.user_arn = success.Arn!;
-      wip_info.user_id = success.UserId;
+      wipInfo.user_account = success.Account!;
+      wipInfo.user_arn = success.Arn!;
+      wipInfo.user_id = success.UserId!;
       // Guess at mfa
-      wip_info.probable_user_mfa_device = derive_mfa_serial(wip_info.user_arn!);
+      wipInfo.probable_user_mfa_device = derive_mfa_serial(wipInfo.user_arn);
     });
 
   // It's built - mark as such
-  const info = wip_info as AuthInfo;
+  const info = wipInfo as AuthInfo;
 
   // Make our request to be the role
   let result: Promise<PromiseResult<STS.GetSessionTokenResponse, AWSError>>;
-  if (mfa_info) {
-    mfa_info.SerialNumber =
-      mfa_info.SerialNumber || info.probable_user_mfa_device!; // We cannot get to this stage if
+  if (mfaInfo) {
+    mfaInfo.SerialNumber =
+      mfaInfo.SerialNumber || info.probable_user_mfa_device!; // We cannot get to this stage if
     result = sts
       .getSessionToken({
         DurationSeconds: duration,
-        SerialNumber: mfa_info.SerialNumber,
-        TokenCode: mfa_info.TokenCode
+        SerialNumber: mfaInfo.SerialNumber,
+        TokenCode: mfaInfo.TokenCode
       })
       .promise();
```

Comment:
Then check if `mfaInfo.SerialNumber` is defined, else ``result = sts.getSessionToken().promise();``